stable/traefik/templates/dashboard-ingress.yaml

```diff
@@ -27,4 +27,8 @@ spec:
       - backend:
           serviceName: {{ template "traefik.fullname" . }}-dashboard
           servicePort: 80
+  {{- if .Values.dashboard.ingress.tls }}
+  tls:
+{{ toYaml .Values.dashboard.ingress.tls | indent 4 }}
```

Comment:
Not a huge fan of this. It is not clear or user friendly. It is not obvious to new users what these mean, or what settings to use. 

I would suggest having the `ingress.tls` section of the values.yaml have elements that are parsed into this section here.

Thoughts?

Reply:
I don't see any benefit of mapping the possible sections of tls into the values. `toYaml` gives you the whole freedom of putting everything what's supported in tls into the values. And `toYaml` is widely used in official Helm charts. I see many other helm charts doing it this way, f.e. https://github.com/helm/charts/blob/master/stable/kubernetes-dashboard/templates/ingress.yaml#L43